deny.toml

```diff
@@ -38,3 +39,16 @@ expression = "LicenseRef-ring"
 license-files = [
     { path = "LICENSE", hash = 0xbd0eed23 },
 ]
+
+[[licenses.clarify]]
+# @thebutlah reviewed the license code. It doesn't impose any copyleft
+# restrictions on us, and doesn't seem to impose any restrictions on
+# distribution. As long as we don't put any other copyleft code in
+# orb-software, we won't run afoul of any license restrictions AFAICT. We
+# should seek to reimplement pyftdi in rust using nusb if we can, though.
```

Comment:
we could try that https://github.com/tanriol/libftdi1-sys 🤔

Reply:
That library is LGPL, so we cannot use it.